### PR TITLE
Vote index fix

### PR DESCRIPTION
**Pull request by MiltonTulli**
# Description

- Fix vote index issues. 
- Fix SidebarInfoCardWrapper undefined braking issue
- Remove enforcedbinaryguild code
- Adding a default "For" option in the actionsbuilder
- Ensure that we display the 'against' option (votable  & non-editable ). For more details [read @mprasanjith comment](https://github.com/DXgovernance/DAVI/issues/20#issuecomment-1183232075)
- Adding a minimum validation when creating proposal
- IPFS issue: catch ipfs error. Fix issue ipfs trying to create more than one instance
- Minor fixes 

Closes https://github.com/DXgovernance/DAVI/issues/63
Closes https://github.com/DXgovernance/DAVI/issues/98
Closes https://github.com/DXgovernance/DAVI/issues/20
Closes https://github.com/DXgovernance/DAVI/issues/65
Closes https://github.com/DXgovernance/DAVI/issues/60

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] My code follows the existing style of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any UI changes have been tested and made responsive for mobile views

### CODE REVIEW

File: src/hooks/Guilds/useVoteSummary.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function useVoteSummary(
     if (totalVotes && totalLocked) {
       const newVotes = [];
       // Skipping the first and showing results from 2nd item (idx 1) onwards. Same case as useVotingResults. TODO: investigate why?
```
Comment (dcrescimbeni) [inline src/hooks/Guilds/useVoteSummary.ts:25]: This comment is no longer relevant

Comment (MiltonTulli) [reply src/hooks/Guilds/useVoteSummary.ts]: It is not! removed

**Comment by MiltonTulli** [comment]
> I'm having a bug where it won't let me create a proposal.
> 
> To reproduce:
> 
> 1. Create a new option and add an action.
> 2. Delete the "For" option
> 3. Try to submit the proposal
> 
> It works OK if I replicate the same scenario in the `develop` branch

I followed your steps but couldn't replicate it. Tested more than 5 times for each guild and working well. Can you provide more details?

https://user-images.githubusercontent.com/25844967/179437893-3bfa2e17-f512-40b5-8ec6-2bbac55ead13.mp4

**Review by rossneilson — APPROVED** [review]
Awesome work Milton!!!!

**Comment by rossneilson** [comment]
> I'm having a bug where it won't let me create a proposal.
> 
> To reproduce:
> 
> 1. Create a new option and add an action.
> 2. Delete the "For" option
> 3. Try to submit the proposal
> 
> It works OK if I replicate the same scenario in the `develop` branch

I am also having replication issues, did you run `yarn` before since we have new packages?

Also important to note that goerli will not work anymore

**Comment by dcrescimbeni** [comment]
I can't reproduce it now after a reboot. It was something on my end.

Awesome PR!

**Pull request merged**